tests/test_learning_path.py

```diff
@@ -0,0 +1,66 @@
+from collections import OrderedDict
+import domain.tutoringModel.generate_learning_path as LP
+import pytest
+
+generate_learning_path = LP.GenerateLearningPath()
```

Comment:
Better use within the test itself instead of a global variable. It should also better be an object with name learning_path